app/scripts/controllers/detect-tokens.js

```diff
@@ -172,6 +178,20 @@ export default class DetectTokensController {
               iconUrl,
               aggregators,
             } = tokenList[nonZeroTokenAddress];
+
+            this._trackMetaMetricsEvent({
+              event: 'Token Detected',
```

Comment:
Q: Should we be adding these event names to `EVENT_NAMES` in `shared/constants/metametrics.js?

cc: @brad-decker

Reply:
@digiwand -- That's a good idea and perhaps a practice we should start doing now even though it isn't widely used yet.

Reply:
While we are on the topic of adding constants, shall I go ahead and add constants for `source` property values? Currently, we have 8 values assigned for the `source` property.